src/Native/Runtime/CommonMacros.h

```diff
@@ -10,6 +10,7 @@
 #elif defined(_TARGET_X86_)
 #elif defined(_TARGET_ARM_)
```

Comment:
I would delete this ifdef. These kind of ifdefs just cause busy work when porting to new architecture.

Reply:
Done